src/analysis/sampling/CHRR/chrrSampler.m

```diff
@@ -1,45 +1,40 @@
-function [samples, roundedPolytope, minFlux, maxFlux] = chrrSampler(model, numSkip, numSamples, toRound, roundedPolytope, minFlux, maxFlux, useFastFVA)
-% Generate uniform random flux samples with CHRR
-% Coordinate Hit-and-Run with Rounding
+function [samples,roundedPolytope] = chrrSampler(model,numSkip,numSamples,toRound,roundedPolytope,minFlux,maxFlux)

```

Comment:
If it is necessary to change the header (which is strongly discouraged), please make sure, that any tutorials and tests are updated accordingly. 
From what I see in the code, this change essentially removed the information in `P.A_eq` and `P.b_eq`. Is that intended? As outputs might change by this alteration.